### PR TITLE
Fixes to merge errors, activated instagram proccesor and fixed errors with ask command

### PR DESCRIPTION
**Pull request by Wikijito7**
Fixed some errors that ocurred after merge, fixed some errors with ask command and activated instagram processor.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -38,9 +39,11 @@ async def on_ready():
     log.info('{0.user} is alive!'.format(bot))
     if is_debug_mode():
         log.debug(">> Debug mode is ON")
+        log.getLogger().setLevel(log.DEBUG)
```
Comment (javdc) [inline main.py:41]: Put this before the log.debug line, or else it won't be printed

File: main.py [hunk 2]
```diff
@@ -224,8 +227,8 @@ async def poll(ctx: Context, *args: str):
 
 
 @bot.command(aliases=["a", "preguntar", "pr"])
-async def ask(ctx: Context, *args: str):
-    text = " ".join(args)
+async def ask(ctx: Context, *, args: str = None):
```
Comment (javdc) [inline main.py:230]: Python magic?

Comment (Wikijito7) [inline main.py]: yes

File: main.py [hunk 3]
```diff
@@ -224,8 +227,8 @@ async def poll(ctx: Context, *args: str):
...
+    text = " ".join(args) if args else ""
```
Comment (javdc) [inline main.py]: Better to just use:

```python
async def ask(ctx: Context, *, text: str = ""):
```

**Pull request merged**